inc/admin.php

```diff
@@ -80,7 +80,17 @@ function settings_screen() {
 	}
 ?>
 <div class="wrap">
-<?php if ( ! empty( $errors ) ) : ?>
+
+	<?php if ( ads_txt_exists() ) : ?>
```

Comment:
can we remove this server side check, and instead start with the div hidden, then call the javascript check on load?

Reply:
Yup! Refactoring it to be a JS only check.